src/niswitch/metadata/functions_addon.py

```diff
@@ -89,9 +89,17 @@
 
 # Default values for method parameters
 function_default_value = {
-    'InitWithTopology':  { 'parameters': { 1: { 'default_value': 'Configured Topology', },
-                                           2: { 'default_value': False, },
-                                           3: { 'default_value': False, },
-                                           4: { 'default_value': '', }, }, },
-}
+    'InitWithTopology':         { 'parameters': { 1: { 'default_value': 'Configured Topology', },
+                                                  2: { 'default_value': False, },
+                                                  3: { 'default_value': False, },
+                                                  4: { 'default_value': '', }, }, },
+    'ConfigureScanList':        { 'parameters': { 2: { 'default_value': 'enums.ScanMode.BREAK_BEFORE_MAKE', 'print_verbatim':True},},},
```

Comment:
You don't need this new piece of redundant metadata 'print_verbatim'. Rather, you can infer that you need to do this by looking at parameter['enum'] is not None.